### PR TITLE
CR 1.5

### PR DESCRIPTION
**Pull request by NoahTheDuke**
Automatically generated from the file given to me by the Nisei Rules Team.

### CODE REVIEW

File: pack/23s.json [hunk 1]
```diff
@@ -110,9 +110,9 @@
         "quantity": 3,
         "side_code": "runner",
         "strength": 1,
-        "stripped_text": "2 credits: Break all subroutines. Use this ability only by spending credits from stealth cards. 1 credit: +1 strength.",
+        "stripped_text": "Interface -> 2click: Break all subroutines. Use this ability only by spending credits from stealth cards. 1click: +1 strength.",
```
Comment (gereons) [inline pack/23s.json:113]: Shouldn't this (and the other costs for Dai V be `[credit]`?

File: pack/cc.json [hunk 2]
```diff
@@ -111,9 +111,9 @@
         "quantity": 3,
         "side_code": "runner",
         "strength": 0,
-        "stripped_text": "If you have at least 2 link, the memory cost of Crowbar is 0, even if it is not installed. Crowbar has +1 strength for each installed icebreaker. trash: Break up to 3 code gate subroutines.",
+        "stripped_text": "If you have at least 2{link}, the memory cost of this program is 0 mu, even if it is not installed. This program gets +1 strength for each installed icebreaker. Interface -> trash: Break up to 3 code gate subroutines.",
```
Comment (gereons) [inline pack/cc.json:114]: `stripped_text` shouldn't contain any braces/brackets

File: pack/cc.json [hunk 3]
```diff
@@ -111,9 +111,9 @@
...
         "stripped_title": "Crowbar",
-        "text": "If you have at least 2[link], the memory cost of Crowbar is 0, even if it is not installed.\nCrowbar has +1 strength for each installed <strong>icebreaker</strong>.\n[trash]: Break up to 3 <strong>code gate</strong> subroutines.",
+        "text": "If you have at least 2{link}, the memory cost of this program is 0[mu], even if it is not installed.\nThis program gets +1 strength for each installed <strong>icebreaker</strong>.\nInterface -> <strong>[trash]:</strong> Break up to 3 <strong>code gate</strong> subroutines.",
```
Comment (gereons) [inline pack/cc.json:116]: Shouldn't this be `[link]` instead of `{link}`? (not only here, but on other cards as well)

File: pack/23s.json (hunk 1 above)
Comment (NoahTheDuke) [reply pack/23s.json:113]: Looks like a bug from Nisei. I'll let them know and fix it here.

File: pack/cc.json (hunk 2 above)
Comment (NoahTheDuke) [reply pack/cc.json:114]: Oops, missed link when processing the brackets.

File: pack/cc.json (hunk 3 above)
Comment (NoahTheDuke) [reply pack/cc.json:116]: Thanks, will fix

**Review by plural — APPROVED** [review]

**Comment by plural** [comment]
oh, actually, you have some json formatting to tidy up first.

**Pull request merged**